js/adapt-contrib-slider.js

```diff
@@ -371,10 +397,13 @@ define(function(require) {
 
         // according to given item index this should make the item as selected
         selectItem: function(itemIndex) {
+            this.$el.a11y_selected(false);
             _.each(this.model.get('_items'), function(item, index) {
                 item.selected = (index == itemIndex);
                 if(item.selected) {
                     this.model.set('_selectedItem', item);
+                    this.$('.confidenceSlider-scale-number[data-id="'+(itemIndex+1)+'"]').a11y_selected(true);
```

Comment:
Oh, cool, must have been looking at the wrong version. I'll remove it. two seconds.